### ISSUE TITLE
Official Fediverse account

### ISSUE THREAD

**Issue by CyberTailor** [gridcoin-community/Gridcoin-Tasks#257]
With Twitter burning to flames, it would be nice to have an official presence in the Fediverse (e.g. Mastodon).

**Comment by CyberTailor** [comment]
Most of fedi is anti-cryptocurrency though, however Gridcoin is not evil, so fedizens might tolerate it.

**Comment by div72** [comment]
Agreed, I would love to hear some opinions on this.

P.S. Who currently manages the Twitter? (@Pythonix ?)

**Comment by Pythonix** [comment]
@div72 Yes, it is mostly me who proposes and posts tweets for Twitter. If the community is in favour of a Mastodon account, we can create an account there as well.

**Comment by Weber462** [comment]
Is this something that need a vote, or can someone just start an account?

**Comment by CyberTailor** [comment]
@TheSCIniative can be found at https://mastodon.social/@the_sci

**Comment by CyberTailor** [comment]
[Buffer](https://buffer.com/mastodon) could be used to publish on multiple social media websites, although its free plan doesn't allow much

**Comment by CyberTailor** [comment]
Threads promises to support Fediverse follows by the late 2024
https://www.theverge.com/2024/1/11/24035256/a-peek-at-the-threads-activitypub-roadmap

**Comment by Frigyes06** [comment]
I would volunteer to run the account. I'll look for servers later today.

**Comment by Frigyes06** [comment]
General:

mastodon.social/about
- The IG mastodon server, moderation not always stellar

mstdn.social/about
- general server

mas.to/about
- general server

Tech focused:

https://mast.hpc.social/about
- High Performance Computing oriented server

https://noc.social/about
- Technical server geared more toward sysadmins and the like

Misc:

https://sonomaca.social/about
- Geographical server administered in part by me, so I can guarantee it wouldn't be banned just bc crypto.


I'd recommend picking one of the tech servers, as the general purpose ones get bad rep for poor moderation and get blocked easily. However there would be more local reach on big servers. I'm open to discussion.

**Comment by CyberTailor** [comment]
If I'd were to choose a server, I'd prefer science-focused instances.

**Comment by Frigyes06** [comment]
> If I were to choose a server, I'd prefer science-focused instances.

@CyberTailor Any examples?

**Comment by Frigyes06** [comment]
https://fosstodon.org/about
- also tech related, FOSS focus

https://fediscience.org/about
- Geared towards academia

**Comment by CyberTailor** [comment]
There's a list of Mastodon-ish instances, plenty to choose from. I'd also talk with instance admins would they be okay with such profile on their server before registration — some of them understandibly don't tolerate everything related to "web3".

https://instances.social/list

**Comment by Frigyes06** [comment]
> Some of them understandibly don't tolerate everything related to "web3".

GridCoin has nothing to do with web3?

**Comment by Frigyes06** [comment]
https://mastodon.social/@gridcoin
